grid/recruitment/list-candidates/maps/greenhouse.suma

```diff
@@ -0,0 +1,49 @@
+profile = "recruitment/list-candidates@1.0"
+provider = "greenhouse"
+
+map ListCandidates {
+  http GET "/v1/candidates" {
+    security "apiKeyBasicAuth"
+    
+
+    request {
+      query {
+        job_id = input.jobId
+        page = input.page
+        per_page = input.limit
+      }
+    }
+
+
+    response 200 "application/json" {
+      map result {
+        candidates = body.map(candidate => {
+        const name = `${candidate.first_name} ${candidate.last_name}`
+        const email = candidate.email_addresses[0] ? candidate.email_addresses[0].value : ''
+        return {
+          id: JSON.stringify(candidate.id),
+          jobId: input.jobId,
+          name: name,
+          email:email
+        }
+        })
+      }
+    }
+
+    response 401 "application/json" {
+      map error {
+        title = 'Unauthenticated'
+        detail = body.message
+        code = 'Unauthenticated'
+      }
+    }
+
+    response 404 "application/json" {
+      map error {
+        title = 'Not Found'
```

Comment:
"Not Found" - presumably the "Job" wasn't found - please add this to the error

(The error message may appear without context to the developer as part of some larger workflow - "Not found" doesn't immediately tell one where to look)

Reply:
I see!

Yes, exactly - in that case, feel free to make additional API calls. The map for BreezyHR does the same: https://github.com/superfaceai/station/blob/main/grid/recruitment/list-candidates/maps/breezy-hr.suma#L151